os-list.csv

```diff
@@ -2,6 +2,7 @@ centos,8
 centos,7
 debian,buster
 debian,stretch
+debian,bullseye
```

Comment:
After adding this line I only executed ./update_dockerfiles

Reply:
great